### ISSUE TITLE
Test "aria-expanded" for mobile menu toggle

### ISSUE THREAD

**Issue by kimisgold** [omeka/theme-thanksroy#60]
Thanks Roy's mobile menu toggle button now uses the `aria-expanded` attribute. Whenever the button receives focus, it should announce to screen readers whether the menu is expanded or collapsed. Test that this is true with VoiceOver, NVDA, and JAWS.

**Comment by katknow** [comment]
I'm getting this error:
`InvalidArgumentException: Could not find file css/public.css! in /var/www/html/katknow/OmekaTesting/application/libraries/globals.php:1289
Stack trace:
#0 /var/www/html/katknow/OmekaTesting/application/libraries/globals.php(1248): web_path_to('css/public.css')
#1 /var/www/html/katknow/OmekaTesting/application/libraries/globals.php(1194): src('css/public.css', 'css', 'css', '3.0.3')
#2 /var/www/html/katknow/OmekaTesting/application/libraries/globals.php(1093): css_src('public', 'css', '3.0.3')
#3 /var/www/html/katknow/OmekaTesting/application/libraries/globals.php(1089): queue_css_file('public', 'all', false, 'css', '3.0.3')
#4 /var/www/html/katknow/OmekaTesting/themes/ThanksRoy/common/header.php(32): queue_css_file(Array)
#5 /var/www/html/katknow/OmekaTesting/application/libraries/Omeka/View.php(114): include('/var/www/html/k...')
#6 /var/www/html/katknow/OmekaTesting/application/libraries/Zend/View/Abstract.php(889): Omeka_View->_run('/var/www/html/k...')
#7 /var/www/html/katknow/OmekaTesting/application/libraries/Zend/View/Helper/Partial.php(109): Zend_View_Abstract->render(NULL)
#8 /var/www/html/katknow/OmekaTesting/application/libraries/Zend/View/Abstract.php(350): Zend_View_Helper_Partial->partial('common/header.p...', Array)
#9 /var/www/html/katknow/OmekaTesting/application/libraries/globals.php(1674): Zend_View_Abstract->__call('partial', Array)
#10 /var/www/html/katknow/OmekaTesting/application/libraries/globals.php(1688): common('header', Array)
#11 /var/www/html/katknow/OmekaTesting/themes/ThanksRoy/index.php(1): head(Array)
#12 /var/www/html/katknow/OmekaTesting/application/libraries/Omeka/View.php(114): include('/var/www/html/k...')
#13 /var/www/html/katknow/OmekaTesting/application/libraries/Zend/View/Abstract.php(889): Omeka_View->_run('/var/www/html/k...')
#14 /var/www/html/katknow/OmekaTesting/application/libraries/Zend/Controller/Action/Helper/ViewRenderer.php(912): Zend_View_Abstract->render(NULL)
#15 /var/www/html/katknow/OmekaTesting/application/controllers/IndexController.php(16): Zend_Controller_Action_Helper_ViewRenderer->renderScript('index.php')
#16 /var/www/html/katknow/OmekaTesting/application/libraries/Zend/Controller/Action.php(516): IndexController->indexAction()
#17 /var/www/html/katknow/OmekaTesting/application/libraries/Zend/Controller/Dispatcher/Standard.php(308): Zend_Controller_Action->dispatch('indexAction')
#18 /var/www/html/katknow/OmekaTesting/application/libraries/Zend/Controller/Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#19 /var/www/html/katknow/OmekaTesting/application/libraries/Zend/Application/Bootstrap/Bootstrap.php(106): Zend_Controller_Front->dispatch()
#20 /var/www/html/katknow/OmekaTesting/application/libraries/Zend/Application.php(384): Zend_Application_Bootstrap_Bootstrap->run()
#21 /var/www/html/katknow/OmekaTesting/application/libraries/Omeka/Application.php(73): Zend_Application->run()
#22 /var/www/html/katknow/OmekaTesting/index.php(23): Omeka_Application->run()
#23 {main}`

I also noticed that on the back end even after I pull and it says I'm up to date on the back end it still say I'm behind an on updates--is that related?

**Comment by katknow** [comment]
This worked in each of the listed screenreaders
